server/src/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -529,14 +545,14 @@ private boolean findCapabilityForService(final Map serviceCapabilitystList, fina
         if (serviceCapabilitystList != null && !serviceCapabilitystList.isEmpty()) {
             final Iterator iter = serviceCapabilitystList.values().iterator();
             while (iter.hasNext()) {
-                final HashMap<String, String> currentCapabilityMap = (HashMap<String, String>) iter.next();
+                final HashMap<String, String> currentCapabilityMap = (HashMap<String, String>)iter.next();
```

Comment:
Please type to the ``Map`` interface rather than ``HashMap`` to reduce coupling.